Timhowie okta 441197 (#2885)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/openidconnect/submit.md

* **What type of sign-in flows do you support?** &mdash; choose either **IdP initiated** or **SP initiated**, or both.

> * **What OIDC mode(s) does your application support?** &mdash; choose either **SPA** (for Single-Page Application) or **Web**.

Reviewer: ```suggestion
* **Which OIDC mode(s) does your application support?**: Choose either **SPA** (for Single-Page Application) or **Web**.
```
